src/test/java/org/mockito/configuration/MockitoConfiguration.java

```diff
@@ -0,0 +1,12 @@
+package org.mockito.configuration;
+
+/**
+ * This is to clean runners between tests.
+ */
+public class MockitoConfiguration extends DefaultMockitoConfiguration {
```

Comment:
Commandline argument? I don't get it. Could you give an example where is the problem?